File: cap11/cap11-backend/src/main/java/fr/eni/ep3jasp/angular/backend/repositories/CuentaRepository.java
package fr.eni.ep3jasp.angular.backend.repositories;

import fr.eni.ep3jasp.angular.backend.domain.Cuenta;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.rest.core.annotation.RepositoryRestResource;

@RepositoryRestResource
public interface CuentaRepository extends JpaRepository<Cuenta, Long> {
}

File: cap18/cap18.mvc/src/main/kotlin/fr/eni/ep3jasp/cap05/kotlin/spring5/mvc/app/repositories/CuentaRepository.kt
package fr.eni.ep3jasp.cap05.kotlin.spring5.mvc.app.repositories

import fr.eni.ep3jasp.cap05.kotlin.spring5.mvc.app.domain.Cuenta
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import java.util.*

// tag::sbmvckt10[]
@Repository
interface CuentaRepository : JpaRepository<Cuenta, UUID>
// end::sbmvckt10[]